Request/SiteRequestContext.php

```diff
@@ -19,14 +20,19 @@
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class SiteRequestContext extends RequestContext
+class SiteRequestContext extends RequestContext implements SiteRequestContextInterface
```

Comment:
this is a BC break and cannot be merged into 3.x

we could do this without the interface in `3.x` and throw a deprecation, that we need to use the new interface in the next major!